Source/JavaScriptCore/runtime/ArrayBuffer.cpp

```diff
@@ -652,5 +652,17 @@ ASCIILiteral errorMessageForTransfer(ArrayBuffer* buffer)
     return "Cannot transfer an ArrayBuffer whose backing store has been accessed by the JavaScriptCore C API"_s;
 }
 
+std::optional<ArrayBufferContents> ArrayBufferContents::fromDataSpan(std::span<const uint8_t> data)
+{
+    void* buffer = Gigacage::tryMalloc(Gigacage::Primitive, data.size_bytes());
+    if (!buffer)
+        return std::nullopt;
+
+    static_assert(sizeof(std::span<const uint8_t>::element_type) == 1);
```

Comment:
I don't think this is necessary.  If sizeof(uint8_t) isn't 1, you're going to have big problems.